airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -1847,6 +1847,8 @@ definitions:
       type:
         type: string
         enum: [RemoveFields]
+      condition:
```

Comment:
let's add a description, title, interpolation_context, and examples. [you can take a look at RecordFilter.condition for a template](https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml#L1786-L1803)

Reply:
added